neuro_san/service/grpc/grpc_agent_server.py

```diff
@@ -40,12 +41,9 @@
 # This is per the lifetime of the server (before it kills itself).
 DEFAULT_REQUEST_LIMIT: int = 1000 * 1000
 
-# A space-delimited list of http metadata request keys to forward to logs/other requests
-DEFAULT_FORWARDED_REQUEST_METADATA: str = "request_id user_id"
-
 
 # pylint: disable=too-many-instance-attributes
-class AgentServer:
+class GrpcAgentServer(AgentServer):
```

Comment:
Rename class for clarity.
Also derive from the AgentServer interface so that HttpSideCar can access this class via that interface for its stop() method.